packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -221,8 +222,8 @@ QUnit.module('collapsible groups', moduleSetup, () => {
         $headers.eq(1).trigger('dxclick');
         $headers.eq(1).trigger('dxclick');
         assert.equal(isRenderer(instance.option('focusedElement')), !!config().useJQuery, 'focusedElement is correct');
-        assert.ok($items.eq(2).hasClass('dx-state-focused'), 'first item of the second group is focused');
-        assert.notOk($items.eq(0).hasClass('dx-state-focused'), 'first item of the first group lost focus');
+        assert.ok($items.eq(2).hasClass(`${LIST_FOCUS_STATE_CLASS}`), 'first item of the second group is focused');
```

Comment:
We don't need string building here. Write just variable name.

Reply:
[cebf195](https://github.com/DevExpress/DevExtreme/pull/27230/commits/cebf1951fc71d84308fbd2bc956dd71c40c4634f)